chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -385,11 +385,13 @@ void BraveRenderViewContextMenu::CopyTextFromImage() {
 
 #if BUILDFLAG(ENABLE_AI_CHAT)
 bool BraveRenderViewContextMenu::IsAIChatEnabled() const {
-  return ai_chat::IsAIChatEnabled(GetProfile()->GetPrefs()) &&
+  return !params_.selection_text.empty() &&
+         ai_chat::IsAIChatEnabled(GetProfile()->GetPrefs()) &&
          brave::IsRegularProfile(GetProfile()) &&
          GetProfile()->GetPrefs()->GetBoolean(
              ai_chat::prefs::kBraveAIChatContextMenuEnabled) &&
-         !params_.selection_text.empty();
+         ai_chat::AIChatTabHelper::FromWebContents(source_web_contents_) &&
+         !IsInProgressiveWebApp();
```

Comment:
This is the fix for the crash

Reply:
Do we still need the other check?
`!params_.selection_text.empty()`

ex: should it be:
```
!params_.selection_text.empty() &&
!IsInProgressiveWebApp();
```